src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -34,58 +33,75 @@ import {
-    settings: '',
-    minimize: 0,
-    window: '',
-    model: 'simple'
-  };
-  const model = { ...defaults, ...props.model };
-  const handleChange = createChangeHandler(props.onChange, model);
-  const handleSelectChange = createSelectHandler(handleChange);
-  const handleTextChange = createTextHandler(handleChange);
-  const handleNumberChange = createNumberHandler(handleChange);
+
+  const model = { ...DEFAULTS, ...props.model };
   const modelOptions = [
     {
-      label: intl.formatMessage({ id: 'tsvb.movingAverage.modelOptions.simpleLabel', defaultMessage: 'Simple' }),
-      value: 'simple'
+      label: intl.formatMessage({
+        id: 'tsvb.movingAverage.modelOptions.simpleLabel',
+        defaultMessage: 'Simple',
+      }),
+      value: MODEL_TYPES.UNWEIGHTED,
     },
     {
-      label: intl.formatMessage({ id: 'tsvb.movingAverage.modelOptions.linearLabel', defaultMessage: 'Linear' }),
-      value: 'linear'
+      label: intl.formatMessage({
+        id: 'tsvb.movingAverage.modelOptions.linearLabel',
+        defaultMessage: 'Linear',
+      }),
+      value: MODEL_TYPES.WEIGHTED_LINEAR,
     },
     {
       label: intl.formatMessage({
-        id: 'tsvb.movingAverage.modelOptions.exponentiallyWeightedLabel', defaultMessage: 'Exponentially Weighted' }),
-      value: 'ewma'
+        id: 'tsvb.movingAverage.modelOptions.exponentiallyWeightedLabel',
+        defaultMessage: 'Exponentially Weighted',
+      }),
+      value: MODEL_TYPES.WEIGHTED_EXPONENTIAL,
     },
     {
-      label: intl.formatMessage({ id: 'tsvb.movingAverage.modelOptions.holtLinearLabel', defaultMessage: 'Holt-Linear' }),
-      value: 'holt'
+      label: intl.formatMessage({
+        id: 'tsvb.movingAverage.modelOptions.holtLinearLabel',
+        defaultMessage: 'Holt-Linear',
+      }),
+      value: MODEL_TYPES.WEIGHTED_EXPONENTIAL_DOUBLE,
     },
     {
-      label: intl.formatMessage({ id: 'tsvb.movingAverage.modelOptions.holtWintersLabel', defaultMessage: 'Holt-Winters' }),
-      value: 'holt_winters'
+      label: intl.formatMessage({
+        id: 'tsvb.movingAverage.modelOptions.holtWintersLabel',
+        defaultMessage: 'Holt-Winters',
+      }),
+      value: MODEL_TYPES.WEIGHTED_EXPONENTIAL_TRIPLE,
     }
   ];
-  const minimizeOptions = [
-    { label: 'True', value: 1 },
-    { label: 'False', value: 0 }
-  ];
+
+  const handleChange = createChangeHandler(props.onChange, model);
+  const handleSelectChange = createSelectHandler(handleChange);
+  const handleNumberChange = createNumberHandler(handleChange);
+
   const htmlId = htmlIdGenerator();
-  const selectedModelOption = modelOptions.find(option => {
-    return model.model === option.value;
-  });
-  const selectedMinimizeOption = minimizeOptions.find(option => {
-    return model.minimize === option.value;
-  });
+  const selectedModelOption = modelOptions.find(({ value }) => model.model_type === value);
+
+  const multiplicativeOptions = [
+    { label: 'True', value: true },
```

Comment:
Should we localize True/ False values  ?

Reply:
Localized